src/ngraph/frontend/onnx_import/onnx.hpp

```diff
@@ -46,35 +46,33 @@ namespace ngraph
         /// \param[in]  domain   The domain the ONNX operator is registered to
         ///                      If not set default domain "ai.onnx" is used
         ///
-        /// \return     True if operator is supported, False otherwise.
+        /// \return     True if operator is supported, False otherwise
         NGRAPH_API
         bool is_operator_supported(const std::string& op_name,
                                    std::int64_t version,
                                    const std::string& domain = "ai.onnx");
 
-        /// \brief      Import and convert an ONNX model from stream to nGraph function.
-        ///             The function translated serialized ONNX model to nGraph function.
+        /// \brief      Import and convert an ONNX model from stream to an nGraph Function representation.
         ///             The serialized ONNX model is read from input stream.
         ///
         /// \note       If parsing stream fails or ONNX model contains not supported ops
         ///             ngraph_error exception can be thrown.
         ///
         /// \param[in]  stream      The input stream (e.g. file stream, memory stream, etc)
         ///
-        /// \return     The function returns a nGraph function representing single output from graph
+        /// \return     The function returns a nGraph Function which represents single output from the created graph
```

Comment:
"an nGraph"

precede vowel sounds with "an" not "a"

Reply:
done